src/test/regress/spec/isolation_shard_rebalancer_progress.spec

```diff
@@ -171,7 +171,8 @@ step "s7-get-progress"
 		targetport,
 		target_shard_size,
 		progress,
-		operation_type
+		operation_type,
+		source_lsn >= target_lsn AND source_lsn > '0/0' as lsn_sanity_check
```

Comment:
Let's split these two checks. Otherwise debugging when they fail is quite difficult.